.github/workflows/run_doublet-detection.yml

```diff
@@ -39,14 +40,21 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Download test data in SCE format
+        env:
+          # Use all samples for testing unless this is triggered by a PR, in which case only use a subset of samples for faster testing
+          # A single sample (or combination of samples for multiplexed data) was chosen for each ScPCA project as:
+          #  Unless otherwise stated, the first sample ID in the given project
+          #  For the multiplexed project SCPCP000009, set of samples `SCPCS000133,SCPCS000134,SCPCS000135,SCPCS000136` (this group of samples has 2 associated libraries). Only `SCPCS000133` is specifed in the SAMPLES string here.
+          #   For the project SCPCP000011, the sample SCPCS000435 was specifically selected as an edge case sample with <10 cells
+          SAMPLES: ${{ github.event_name != 'pull_request' && 'all' || 'SCPCS000001,SCPCS000024,SCPCS000050,SCPCS000101,SCPCS000124,SCPCS000133,SCPCS000168,SCPCS000216,SCPCS000246,SCPCS000250,SCPCS000298,SCPCS000435,SCPCS000481,SCPCS000484,SCPCS000490,SCPCS000502,SCPCS000514,SCPCS000616,SCPCS000632,SCPCS000758' }}
+
         run: |
           conda activate openscpca-doublet-detection
```

Comment:
Is this `conda` line still required? I ask for two reasons: we did update the container (which still works in general here, yay!) but I noticed that it wasn't needed in the next step, which makes me think we maybe don't need it any more? Or maybe because there is an explicit call to `bash` within that chunk, it works out and does call `.bashrc` as it launches the script. 

Or, as I think about it, it is probably the fact that you are just using R there...

Reply:
Yep, it's b/c R+python in benchmark step, but just R in scpca step. Your comment here suggests that a comment in the GHA couldn't hurt!

Reply:
(but I suppose the benchmark step isn't in this PR...)